.github/ISSUE_TEMPLATE/FEATURE.md

```diff
@@ -0,0 +1,23 @@
+---
+name: 🚀 Feature request
```

Comment:
I think maybe larger feature requests can go into https://github.com/TykTechnologies/tyk-operator/discussions

If it's a small feature request, We should enhance this template to encourage feature requests to follow the BDD format.

sample:
```
Feature: Managing http APIs
  In order to control the Tyk Gateway
  As a developer
  I need to be able to create, update and delete API resources
  And the gateway should reconcile accordingly
  Scenario: Create a keyless api
    Given there is a ./custom_resources/httpbin.keyless.apidefinition.yaml resource
    When i request /httpbin/get endpoint
    Then there should be a 200 http response code

  Scenario: Update an api from keyless to auth token
    Given there is a ./custom_resources/httpbin.keyless.apidefinition.yaml resource
    When i update a ./custom_resources/httpbin.apikey.apidefinition.yaml resource
      And i request /httpbin/get endpoint
    Then there should be a 401 http response code

  Scenario: Delete an API
    Given there is a ./custom_resources/httpbin.keyless.apidefinition.yaml resource
    When i delete a ./custom_resources/httpbin.keyless.apidefinition.yaml resource
      And i request /httpbin/get endpoint
    Then there should be a 404 http response code

  Scenario: Transform xml to json
    Given there is a ./../config/samples/httpbin_transform.yaml resource
    When i request /httpbin-transform/xml endpoint
    Then there should be a "Content-Type: application/json" response header
```

Reply:
RFCs via the discussions feature - yep, love it 💯  will add a note and a link to direct larger changes in that direction.
Will add a slimmed-down BDD skeleton to the template as well.

Reply:
f78058b and 063c5cd should resolve.